website/aibot/chunk_utils.py

```
@@ -0,0 +1,682 @@
+from website.aibot.utils import approximate_token_count_char
+
+logger = logging.getLogger(__name__)
+
+
+MAX_TOKENS = 1500
+OVERLAP_LINES = 7
+CHUNK_OVERLAP = 200
+CHUNK_SIZE = 1000
+
+
+import math
```

Ruff (0.12.2)

Redefinition of unused `math` from line 4

Remove definition: `math`

(F811)